src/document/crdt/rht_pq_map.ts

```diff
@@ -57,8 +57,8 @@ export class RHTPQMapNode extends HeapNode<TimeTicket, CRDTElement> {
   /**
    * `remove` removes a value base on removing time.
    */
-  public remove(removedAt: TimeTicket): void {
-    this.getValue().remove(removedAt);
+  public remove(removedAt: TimeTicket): boolean {
```

Comment:
I think TypeScript 4.0 checks more strictly. And this change might fix a bug.

Reply:
https://github.com/yorkie-team/yorkie/blob/69fb07c8dfa35c891c13d98807cf60b78c2247ce/pkg/document/json/rht_pq_map.go#L41-L44